### PR TITLE
Login css

### PR DESCRIPTION
**Pull request by ahmedalmi95**

### CODE REVIEW

**Review by Anies12 — APPROVED** [review]

File: src/controllers/home.js
Comment (tomduggan85) [inline src/controllers/home.js:5]: @AhmedAl-Almi Is it possible to do a more specific check here, for example `if (req.cookies.token) {` or something like that?  If so, that might be safer (to prevent an unrelated request cookie causing the user to go to the login page).  Maybe it's not possible 🤷‍♂️

File: public/css/login.css
Comment (tomduggan85) [inline public/css/login.css:82]: @AhmedAl-Almi It's best if your `transition` styles always specified which attribute you want the browser to transition (so here, it would be `transition: color .2s`). 
This will have better performance (discussed [here](https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x)) because the browser isn't trying to transition things that the user can't see.
That will also prevent you from seeing things animate that you didn't expect to animate (if the user changes the browser zoom or the page layout changes and this element changes in size, you probably don't want to see that change animate)

**Comment by Angham116** [comment]
@AhmedAl-Almi If you've been given an ID's and Classes name for the elements in this file
https://github.com/FACG6/Gaza-Freelancing-Community/blob/a6784f4cf85c6dd52fedce611152745ab0f8d838/src/views/login.hbs
 why not use them here ?

**Review by Angham116 — COMMENTED** [review]
@AhmedAl-Almi If you've been given an ID's and Classes name for the elements in this file
https://github.com/FACG6/Gaza-Freelancing-Community/blob/a6784f4cf85c6dd52fedce611152745ab0f8d838/src/views/login.hbs
 why not use them here ?

**Review by Fatmasiam — APPROVED** [review]

File: src/controllers/home.js
Comment (ahmedalmi95) [reply src/controllers/home.js:5]: Sure you right, but I do not finish from this file I used it to render home just as a test I will return for this code again and I added "jwt" in the browser manually to render home because of the login post #14  not finish. @Anies12  made it #90 but waiting for approval if you can make a review for this branch if you like, please.  :)

File: public/css/login.css
Comment (ahmedalmi95) [reply public/css/login.css:82]: This is a good idea, I like it.

**Comment by ahmedalmi95** [comment]
I used it in the dom file.
I used tags because we have single file CSS for login don't effect on others files.

File: src/controllers/home.js
Comment (tomduggan85) [reply src/controllers/home.js:5]: Sounds good!

**Pull request closed without merging**
